### PR TITLE
apitest recognize Set-Cookie header and set cookie to latter apis

### PR DESCRIPTION
**Pull request by sfwn**
#### What type of PR is this?

bug

#### What this PR does / why we need it:

use `Set-Cookie` header in response instead of CookieJar, and set `Cookie` header for latter apis to simulate login.

### CODE REVIEW

File: modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/meta.go [hunk 1]
```diff
@@ -86,11 +88,8 @@ func writeMetaFile(ctx context.Context, task *spec.PipelineTask, meta *Meta) {
 	}
 	if meta.Resp != nil {
 		kvs.add(metaKeyAPIResponse, jsonOneLine(ctx, meta.Resp))
-	}
-	if meta.CookieJar != nil {
-		if meta.Resp != nil && meta.Resp.Headers != nil && meta.Resp.Headers["Set-Cookie"] != nil {
-			jar, _ := json.Marshal(meta.CookieJar)
-			kvs.add(metaKeyAPICookies, string(jar))
+		if headerSetCookie := meta.Resp.Headers[apitestsv2.HeaderSetCookie]; len(headerSetCookie) > 0 {
```
Comment (kakj-go) [inline modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/meta.go:91]: if Resp is nil,  Resp.Header will npe

File: modules/pipeline/aop/plugins/task/plugins/autotest_cookie_keep_before/plugin.go [hunk 2]
```diff
@@ -44,45 +47,58 @@ func (p *Plugin) Handle(ctx *aoptypes.TuneContext) error {
...
+			rlog.TErrorf(ctx.SDK.Pipeline.ID, ctx.SDK.Task.ID, "failed to unmarshal AUTOTEST_API_GLOBAL_CONFIG, err: %v", err)
 			return err
 		}
 	}
 	if config.Header == nil {
 		config.Header = map[string]string{}
 	}
-	config.Header[autotest_cookie_keep_after.CookieJar] = cookieJar
-	configJson, err := json.Marshal(config)
+	config.Header[apitestsv2.HeaderCookie] = setCookie
```
Comment (kakj-go) [inline modules/pipeline/aop/plugins/task/plugins/autotest_cookie_keep_before/plugin.go:88]: if config user set cookie in header, here you will cover user cookie

File: modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/meta.go (hunk 1 above)
Comment (sfwn) [reply modules/pipeline/pipengine/actionexecutor/plugins/apitest/logic/meta.go:91]: Yes, I will check it.

File: modules/pipeline/aop/plugins/task/plugins/autotest_cookie_keep_before/plugin.go (hunk 2 above)
Comment (sfwn) [reply modules/pipeline/aop/plugins/task/plugins/autotest_cookie_keep_before/plugin.go:88]: Overwrite cookie can implement user switch.

**Review by johnlanni — APPROVED** [review]

**Pull request merged**